spec/pagseguro/payment_request/request_serializer_spec.rb

```diff
@@ -61,125 +22,230 @@
           }
         }
       ]
+      expect(subject.to_xml_params).to match /<receivers>.*<receiver>.*<email>a@example.com<\/email>/m
     end
 
-    it { expect(params).to include('receiver[1].email' => 'a@example.com') }
-    it { expect(params).to include('receiver[1].split.amount' => 10) }
-    it { expect(params).to include('receiver[1].split.feePercent' => 11) }
-    it { expect(params).to include('receiver[1].split.ratePercent' => 12) }
-  end
-
-  context "address serialization" do
-    let(:address) do
-      PagSeguro::Address.new(
-        street: "STREET",
-        state: "STATE",
-        city: "CITY",
-        postal_code: "POSTAL_CODE",
-        district: "DISTRICT",
-        number: "NUMBER",
-        complement: "COMPLEMENT"
+    it 'should serializer sender' do
+      payment.sender = PagSeguro::Sender.new(
+        name: 'Sender',
+        email: 'b@example.com',
+        phone: PagSeguro::Phone.new(area_code: 51, number: 2)
       )
+      expect(subject.to_xml_params).to match /<sender>.*<name>Sender<\/name>.*<email>b@example.com<\/email>.*<phone>.*<areaCode>51<\/areaCode>.*<number>2<\/number>/m
```

Comment:
Because of the final `/m`.

`//` means `%r[]`, I don't know how to use `//m` :P